src/modules/context2d.js

```diff
@@ -641,6 +643,32 @@ import {
       }
     });
 
+    /**
+     * A float specifying the amount of the line dash offset. The default value is 0.0.
+     *
+     * @name lineDashOffset
+     * @default 0.0
+     */
+    Object.defineProperty(this, "lineDashOffset", {
+      get: function() {
+        return this.ctx.lineDashOffset;
+      },
+      set: function(value) {
+        this.ctx.lineDashOffset = value;
+        setLineDash.call(this);
+      }
+    });
+
+    Object.defineProperty(this, "lineDash", {
```

Comment:
Maybe we should also add the "Not HTML API" comment here?

Reply:
Added comments.